slu/tests/test_contextual_slu/test.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import os
+from tqdm import tqdm
+import yaml
+from slu.utils.config import YAMLPromptConfig
+from slu.dev.prompt_setup import setup_prompts
+from slu import constants as const
+import argparse
+
+
+def load_tests(path='tests/test_contextual_slu/test_cases.yaml')-> dict:
+    """
+    Custom function to load test cases (.yaml file)
+    """
+    if not os.path.exists(path):
+        raise Exception(
+            f"Invalid path: {path}"
+        )
+    with open(path, 'r') as fp:
+        test_cases = yaml.load(fp, Loader=yaml.FullLoader)
+    
+    return test_cases
+
+def reset_output(dest)-> None:
+    """
+    Custom function to delete output files. 
+    """
+    if os.path.exists(os.path.join(dest,'prompts.yaml')):
+        os.remove(os.path.join(dest,'prompts.yaml'))
+        
+    if os.path.exists(os.path.join(dest,'missing_prompts.yaml')):
+        os.remove(os.path.join(dest,'missing_prompts.yaml'))
+        
+
+def test_prompt_config()-> None:
+    """
+    Unit tests to evaluate Class YAMLPromptConfig from slu.utils.config
+    """
+    test_cases = load_tests()
```

Comment:
This is the wrong way to pass multiple test cases to the same test. See [this](https://github.com/skit-ai/dialogy-template-simple-transformers/blob/main/slu/tests/test_controller/test_predict_api.py#L114) on how to do it the correct way in pytests.

Reply:
Made the necessary changes